concent_api/concent_api/settings/base.py

```diff
@@ -309,6 +309,9 @@
 # completing verification.
 ADDITIONAL_VERIFICATION_TIME_MULTIPLIER = 2.0
 
+# A global constant defining deadline for submitting SubtaskResultsVerify.
+ADDITIONAL_VERIFICATION_CALL_TIME = 10  # TODO: What is the default in GM?
```

Comment:
please resolve this before merging ;)

Reply:
Yes, there is a separate issue for it in golem-messages which is also added as a blocker to this task (maybe I should mention this in pull request description, I will now) but I that when it is ready then it should be just single line switch.